src/main/java/com/ensiasit/projectx/dto/MemberResponse.java

```diff
@@ -0,0 +1,20 @@
+package com.ensiasit.projectx.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Set;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+@Builder
+public class MemberResponse {
+    private long id;
+
+    private String teamName;
+
+    private Set<String> membersUsernames;
```

Comment:
No need for that, since it is contained in TeamResponse, and it won't be used after membership request